presto-hive-metastore/src/test/java/com/facebook/presto/hive/MockHiveMetastore.java

```diff
@@ -35,7 +36,7 @@
 
     public MockHiveMetastore(MockHiveCluster mockHiveCluster)
     {
-        super(mockHiveCluster, new MetastoreClientConfig());
+        super(mockHiveCluster, new MetastoreClientConfig(), new NoHiveMetastoreAuthentication());
```

Comment:
`NONE`
Do static import of `HiveMetastoreAuthenticationType.NONE`
Same comment for all other places.